content/blog/2023-12-20-changepoint_explore.Rmd

```diff
@@ -9,10 +9,12 @@ tags:
 authors:
   - tara
   - ajoshi
-heroImage: changepoint-hero.jpg
-heroImageThumb: changepoint-thumb.png
+heroImage: blog-lg-changepoint.jpeg
+heroImageThumb: blog-thumb-changepoint.jpeg
```

Comment:
I was confused as to why you were using new file names without adding new files, and how the preview still worked properly...  But now i see some images were added in this commit that went in without a PR: https://github.com/cmu-delphi/www-main/commit/0fb1eb40ecc0995b38247a0b30c0eed66edbcab3 .

You should delete the old `changepoint-hero.jpg` and `changepoint-thumb.png` files that are being replaced here.

Reply:
Oops, there is no preview.  I guess i was thinking of a different PR.

Reply:
We need to get this PR merged before we do a release, because those image file changes that went in without a PR make the blog list a little wonky, and this one fixes it.  To see the wonkiness, this preview link should show it to you with a lil bit of scrolling: https://deploy-preview-919--cmu-delphi-main.netlify.app/blog/

Reply:
Will work on the changes now! But I don't see wonkiness in the preview?

Reply:
on that preview, in the list of blog articles, the changepoint article's "row" is a little too tall....  ¯\\\_(ツ)_/¯